demo/picture.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE HTML>
```

Comment:
@dutchenkoOleg what's the need to add a complete new demo page for `picture` tag use?

Reply:
I created new demo page for my own local testing.

In index demo page you have hardcoded style elements with inheritance from the id selector.  
Also you use custom `load` method for demonstrations of loading process, which overrided  
default load _logic_ with picture tag implementation

Reply:
please make use of the existing demo page, you can add a new CSS rule as per requirement. Also, you ca have multiple instances of Lozad on the same page.

Reply:
Sorry for waiting. Just updated